node/resolvers/session/index.ts

```diff
@@ -98,4 +116,25 @@ export const mutations = {
 
     return true
   },
+
+  savePickupInSession: async (_: any, args: SavePickupArgs, ctx: Context) => {
+    const { address, name } = args
+    const {
+      clients: { session },
+      cookies,
+    } = ctx
+
+    await session.updateSession(
```

Comment:
is `session` guaranteed to exist, and if so, is `updateSession` always a function?

Reply:
yes and yes